SupportBank.Console/InputParser/JsonParser.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using NLog;
+
+namespace SupportBank.Console
+{
+    public class JsonParser : IInputParser
+    {
+        private static readonly ILogger logger = LogManager.GetCurrentClassLogger();
+
+        public List<Transaction> ParseInput(string filePath)
+        {
+            logger.Debug("Parsing JSON file " + filePath);
+            List<Transaction> newTransactions = new List<Transaction>();

```

Comment:
It's ok to leave it like this, but you don't have to specify the type explicitly on the left side, you can do just `var newTransactions = ... `, as long as you specify it on the right side. It's quicker this way, and it's still clean & readable ;)

Reply:
Especially since the IDE will likely tell you the type if you hover over the var